### DISCUSSION TITLE
Random wallpaper. Can i add a blur effect

### DISCUSSION THREAD

**Discussion by JackMcHudson** [rocksdanister/lively#2549] [Q&A]
I'v used the script for random wallpapers on lively. But every time it changes the screen turns black for half a second. Therefore I would like to add a blur effect berween wallpapers. And the same wallpaper keeps looping sometimes. Anyone can help me with this.

**Comment by rocksdanister** [comment]
Regarding the audio volume not working you asked earlier - webpage wallpapers only have ON (anything above 0) and OFF (0) state, so that's probably whats happening.

>  And the same wallpaper keeps looping sometimes

The random algorithm does not checking current running wallpaper so unfortunately sometimes it picks the same wallpaper again, you could write your own random wallpaper algorithm. For example:
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls#random-wallpaper-medium

> Therefore I would like to add a blur effect berween wallpapers.

Not possible currently since each time new wallpaper player instance is created and previous shutdown.
